README.md

```diff
@@ -1 +1,29 @@
 # aepp-showcases
+
```

Comment:
Missing description what this actually is

Reply:
Dang it. Rebase swallowed my README 👎